casepro/statistics/signals.py

```diff
@@ -52,3 +53,24 @@ def record_incoming_labelling(sender, instance, action, reverse, model, pk_set,
     elif action == 'pre_clear':
         for label in instance.labels.all():
             DailyCount.record_removal(day, DailyCount.TYPE_INCOMING, label)
+
+
+@receiver(post_save, sender=CaseAction)
+def record_new_case_action(sender, instance, created, **kwargs):
+    """
+    This is where we keep track of DailyCounts for users within organisations
+    """
+    org = instance.case.org
+    user = instance.created_by
```

Comment:
I'm wondering if we should be tracking the users that are opening the cases, or the users that the cases are assigned to.